### PR TITLE
[TOOLS] Upgrade to gradle 5.2.1

### PR DESCRIPTION
**Pull request by takirala**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,2 +1,3 @@
-org.gradle.daemon=false
```
Comment (takirala) [inline gradle.properties]: i could not find a good argument for having this (either in jira or in git log). I did not see any problem (yet).

File: gradle/quality.gradle [hunk 2]
```diff
@@ -13,7 +12,6 @@ tasks.withType(FindBugs) {
 }
 
 checkstyle {
-    toolVersion = "8.14"
```
Comment (kvish) [inline gradle/quality.gradle]: what was this for?

File: gradle/quality.gradle [hunk 3]
```diff
@@ -1,7 +1,6 @@
 apply plugin: 'findbugs'
 apply plugin: 'checkstyle'
 apply plugin: 'pmd'
-apply plugin: 'jacoco'
```
Comment (kvish) [inline gradle/quality.gradle]: afaik this was for code coverage numbers, any reason we are removing it?

Comment (takirala) [reply gradle/quality.gradle]: Its not removed, this was redundant code : https://github.com/mesosphere/dcos-commons/blob/master/build.gradle#L21

File: gradle/quality.gradle (hunk 2 above)
Comment (takirala) [reply gradle/quality.gradle]: We use the default plugin version bundled, saves us from updating it from time-to-time.

File: tools/create_testing_volumes.py [hunk 4]
```diff
@@ -73,6 +73,7 @@ def mount_volumes():
 export MESOS_WORK_DIR MESOS_RESOURCES
 eval $(sed -E "s/^([A-Z_]+)=(.*)$/\\1='\\2'/" /opt/mesosphere/etc/mesos-slave-common)  # Set up `MESOS_WORK_DIR`.
 eval $(sed -E "s/^([A-Z_]+)=(.*)$/\\1='\\2'/" /opt/mesosphere/etc/mesos-slave)         # Set up `MESOS_RESOURCES`.
+source /opt/mesosphere/etc/mesos-slave-common
```
Comment (takirala) [inline tools/create_testing_volumes.py]: I will remove this part of the diff, this was done when i am experimenting with dcos-launch stuff.

**Comment by takirala** [comment]
Merging this pending CI checks cuz all the checks passed on a previous commit e5560834aa062182966c4b0fca301bd2689b1409 
The last commit in the PR is an irrelevant revert.

**Pull request merged**